### PR TITLE
Display current income rate

### PR DESCRIPTION
**Pull request by joshuaclayton**
What?
=====

This displays the current income rate, which includes both the base
income from resources and a running 10 second average of income from
clicks.

income-rate

### CODE REVIEW

File: src/elm/Model.elm [hunk 1]
```diff
@@ -41,4 +47,33 @@ initial =
...
+            Expirable.expiresIn (Expirable.SecondsRemaining recentlyGeneratedCurrencyWindowInSeconds) amount
+    in
+    { model | recentlyGeneratedCurrency = generatedCurrency :: model.recentlyGeneratedCurrency }
+
+
+clickedIncomeRatePerSecond : Model -> IncomeRate.IncomeRate
+clickedIncomeRatePerSecond { recentlyGeneratedCurrency } =
+    List.map (IncomeRate.IncomeRate << Expirable.value) recentlyGeneratedCurrency
+        |> IncomeRate.sum
+        |> flip IncomeRate.multiply (1 / recentlyGeneratedCurrencyWindowInSeconds)
```
Comment (JoelQ) [inline src/elm/Model.elm]: I don't think you need to flip `multiply`? I think it should be commutative?

**Review by JoelQ — APPROVED** [review]
One comment, otherwise LGTM 👍

File: src/elm/Model.elm [hunk 2]
```diff
@@ -41,4 +47,33 @@ initial =
...
+
+
+recentlyGeneratedCurrencyWindowInSeconds : Float
```
Comment (johnschoeman) [inline src/elm/Model.elm]: is this something that should be living in game config?

**Review by johnschoeman — APPROVED** [review]
LGTM!
one question

**Pull request merged**